### DISCUSSION TITLE
CKE Off Residency

### DISCUSSION THREAD

**Discussion by AnaKhorguani** [opcm/pcm#464] [Q&A]
Hello,

I was wondering which power-down state does CKE Off Residency correspond to? As I found there are around 7 power-down modes, starting from `active standby` down to `self refresh`.  

So when I get `CKE Off Residency: 0%`, I assume that the memory module is in `active standby` state. But, on the other hand, when I see `CKE Off Residency: 95.09%` for example, does this correspond to one specific state (if yes, then which one?), or can the memory module be in several different possible states?

**Comment by ogbrugge** [comment]
@rdementi Can you answer this one?

**Comment by opcm** [comment]
I am not an expert on DRAM power modes but perhaps this doc can help: https://edc.intel.com/content/www/cn/zh/design/ipla/software-development-platforms/client/platforms/alder-lake-desktop/12th-generation-intel-core-processors-datasheet-volume-1-of-2/001/dram-power-management-and-initialization/
